### DISCUSSION TITLE
Dashboard crashing after upgrade to 4.20

### DISCUSSION THREAD

**Discussion by gbhoffman** [nextgenhealthcare/connect#5675] [Q&A] [answered]
I had a happily-running version 3.11 on Windows Servwr 2016, but then I added Microsoft's OpenJDK and upgraded to MirthConnect 4.20. After importing my config file, all my channels seem in place and deployed, but the Mirth Connect Administrator Dashboard errors with a looong list of issues. mcserver seems to be running ok, but without the insight from Dashboard, I don't know what's going on.

Things I've done:
Uninstalled Oracle Java
Rebooted Windows
Checked that both mcserver and MC Admin are using the OpenJDK instance, 17.0.6
Launched MC Admin with bundled and with default Java path
Confirmed PostgreSQL is running and MC is making a connection to it

Any other ideas would be welcome.

Thanks,
Gary

**Comment by pacmano1** [comment, marked as answer]
Did you add the vmoptions required for java9+ that are in the readme?

**Reply by pacmano1** [reply]
Also I prefer Amazon's Openjdk Coretto. Not had  problems with it on the rare occasion mirth is running on a Windows box.

**Comment by gbhoffman** [comment]
Well, finding that Readme file was a hunt, then saving the revised file was a chore, but I finally did it and -- yes, that worked. And the Dashboard is very happy now.

So thanks for the reminder. Someone ought to make that at least as easy as the rest of this program is.

Thanks,
Gary
